.ci/scripts/prepare-benchmarks-env.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# This particular configuration is required to be installed in the baremetal
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+command -v nvm
+
+nvm install node
```

Comment:
This will always just install the latest verison of Node.js. For benchmarks it might be good to either:

- Normally always run on the same version, and only at certain times upgrade
- Or, use the latest LTS version

Currently, the latest version is v12.x and the latest LTS is v10.x. Right now it's probably ok to use v12.x, but when v13.0.0 is released this will now be the latest, but it will be far from stable. So it might not be a good idea to automatically run benchmarks on that.

Alternatively, we should run benchmarks on both the latest and on LTS. It would actually be really interesting to compare those numbers to see if the benchmarks become slower or faster by upgrading to a newer version of Node.js

Reply:
For such, I'd propose to parameterize the script `.ci/scripts/prepare-benchmarks-env.sh` with the node version to be used.

I'll use the node version `v12.10.0`, won't I?

Reply:
Right now the easiest would probably be to change it to `nvm install 'lts/*'`. That should install the latest LTS version... In the future we could then make two benchmark runs, one that uses `lts/*` and another that uses `stable` (which currently would be 10.x and 12.x respectively)

Reply:
You could also hardcode it to `nvm install 12` now if you wanted to

Reply:
Already implemented, https://github.com/elastic/apm-agent-nodejs/pull/1145/commits/a620bd362a7a8589b70bca0eea3f5c3402049446 in conjunction with https://github.com/elastic/apm-agent-nodejs/pull/1145/commits/7f1e26508b2da6658709a605b7b8abe008625c49 and https://github.com/elastic/apm-agent-nodejs/pull/1145/commits/9b75bbdf286b0bf60d70208dc68ae78dda164c95

Reply:
Therefore, if the env variable `NODE_VERSION` exists then it will install that particular version otherwise it would use the `node` default one, aka the latest version.